### PR TITLE
Add mongodb tuning for 6.7

### PR DESCRIPTION
**Pull request by Imaanpreet**

### CODE REVIEW

**Review by psuriset — COMMENTED** [review]
I would remove "some times".

**Review by psuriset — CHANGES_REQUESTED** [review]
user with out good amount of memory cant allocate 20% 
Provide memory guidelines link here.  We can make this recommendation, assuming user followed the memory capacity planning doc.

**Comment by Imaanpreet** [comment]
I have updated the file,please check.

File: docs/satellite-configuration-tuning.rst [hunk 1]
```diff
@@ -229,6 +229,23 @@ Note:
 MongoDb Tuning
 ==============
 
+mongod consumes randomly high memory (up to 1/2 of all RAM) and this aggressive memory usage limits other processes or can cause OOM killers to kill mongod. In order to overcome this situation, tune the cache size by referring the following steps:
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst]: randomly high - Are we sure that is random? If not I would rephrase to "under certain circumstances" or so

OOM killers => OOM killer

File: docs/satellite-configuration-tuning.rst [hunk 2]
```diff
@@ -229,6 +229,23 @@ Note:
...
+
+Update custom-hiera.yaml:
+
+-  Edit /etc/foreman-installer/custom-hiera.yaml and add the entry below inserting the value that is 20% of the physical RAM while keeping in mind the `guidlines <https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7-beta/html/installing_satellite_server_from_a_connected_network/preparing_your_environment_for_installation#hardware_storage_prerequisites`_, in this case, approximately 10GB for a 48GB server::
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst]: I would add non-beta link (remove `/beta/`) even with risk final link will be incorrect

File: docs/satellite-configuration-tuning.rst [hunk 3]
```diff
@@ -229,6 +229,23 @@ Note:
...
+
+  mongodb::server::config_data:
+   storage.wiredTiger.engineConfig.cacheSizeGB: 10
+
+- Run installer to apply changes::
```
Comment (jhutar) [inline docs/satellite-configuration-tuning.rst:241]: Maybe refer some chapter that have this generic step? (so we do not need to change N places when the process changes)

**Review by jhutar — CHANGES_REQUESTED** [review]

File: docs/satellite-configuration-tuning.rst (hunk 1 above)
Comment (Imaanpreet) [reply docs/satellite-configuration-tuning.rst]: I am not sure 100%, so changing according to the suggestions.

File: docs/satellite-configuration-tuning.rst (hunk 2 above)
Comment (Imaanpreet) [reply docs/satellite-configuration-tuning.rst]: Yesterday, the only a beta link was available, today they uploaded the doc for 6.7. In short, I have changed the link.
